src/components/WwwFrame/PromotionalBanner/GlobalPromotionalBannerContentful.vue

```diff
@@ -142,8 +154,23 @@ export default {
 						this.isPromoEnabled = true;
 					}
 				}
+
+				return this.apollo.query({
```

Comment:
Since this is in the result function it won't finish before `created` is called. The experiment assignment could also be fetched by adding the experiment field to the `bannerQuery` above:
```graphql
query bannerQuery {
	contentful {
		entries(contentType: "uiSetting", contentKey: "ui-global-promo")
	}
	experiment(id: "deposit_incentive_banner") @client {
		id
		version
	}
}
```